integration-tests/mybatis/src/main/java/org/apache/camel/quarkus/component/mybatis/it/MybatisResource.java

```diff
@@ -61,4 +77,44 @@ public Response loadComponentMybatisBean() throws Exception {
         LOG.warnf("Could not load [%s] from the Camel context", COMPONENT_MYBATIS_BEAN);
         return Response.status(500, COMPONENT_MYBATIS_BEAN + " could not be loaded from the Camel context").build();
     }
+
```

Comment:
Could you please remove the original auto-generated test ensdpoints `loadComponentMybatis()` and `loadComponentMybatisBean()`?

Reply:
yeah, I will do.

Reply:
Done